tailwind.config.js

```diff
@@ -11,11 +11,28 @@ module.exports = {
       "fira-sans": ["Fira Sans", "sans-serif"],
       "work-sans": ["Work Sans", "sans-serif"],
     },
+  fontSize: {
+      sm: '0.8rem',
+      base: '1rem',
+      xl: '1.25rem',
+      '2xl': '1.563rem',
+      '3xl': '1.953rem',
+      '4xl': '2.441rem',
+      '5xl': '3.052rem',
+      'half': '3.5rem',
+      '3.5xx': '3.5rem',
+    },
```

Comment:
Lo mejor es que esto no se haga por que lo que hace es sobre escribir los fontSize por defecto, lo mejor es agregarlo en extend.

Reply:
https://www.geeksforgeeks.org/how-to-customize-the-font-size-in-tailwind-css/

Reply:
@Phosphorus-M  listo :D